x-pack/plugins/security_solution/common/detection_engine/schemas/common/sorting.ts

```diff
@@ -10,7 +10,22 @@ import type { Either } from 'fp-ts/lib/Either';
 import { capitalize } from 'lodash';
 
 export type SortField = t.TypeOf<typeof SortField>;
-export const SortField = t.string;
+export const SortField = t.union([
+  t.literal('created_at'),
+  t.literal('createdAt'), // Legacy notation, keeping for backwards compatibility
+  t.literal('enabled'),
+  t.literal('execution_summary.last_execution.date'),
+  t.literal('execution_summary.last_execution.metrics.execution_gap_duration_s'),
+  t.literal('execution_summary.last_execution.metrics.total_indexing_duration_ms'),
+  t.literal('execution_summary.last_execution.metrics.total_search_duration_ms'),
+  t.literal('execution_summary.last_execution.status'),
+  t.literal('name'),
+  t.literal('risk_score'),
+  t.literal('riskScore'), // Legacy notation, keeping for backwards compatibility
+  t.literal('severity'),
+  t.literal('updated_at'),
+  t.literal('updatedAt'), // Legacy notation, keeping for backwards compatibility
+]);
```

Comment:
We should not forget to update the docs with this info. I can open a ticket if you haven't already.